src/screens/Display/index.js

```diff
@@ -1,18 +1,36 @@
-import React, {useState} from 'react';
-import {useDispatch} from 'react-redux';
+import React, {useState, useLayoutEffect} from 'react';
+import {connect, useDispatch} from 'react-redux';
 import {View, Text, TextInput, Alert, TouchableOpacity} from 'react-native';
 
 import CalcButton from '../../components/CalcButtons';
 import {styles} from './style';
-import actionsCreator from '../../redux/actions';
+import authAction from '../../redux/auth/actions';
+import historyAction from '../../redux/historyApi/actions';
 import {RULES} from '../../constants/rules';
 import {buttonsCreator} from '../../utils/buttons';
 
-const Display = ({navigation}) => {
+const mapStateToProps = state => {
+  const {user, history} = state;
+  return {user, history};
+};
+
+const Display = ({user, navigation}) => {
   const [userText, setUserText] = useState('');
   const [calcText, setCalcText] = useState('');
   const dispatch = useDispatch();
 
+  useLayoutEffect(() => {
+    navigation.setOptions({
+      headerRight: () => (
+        <TouchableOpacity
+          style={styles.btnLogOut}
+          onPress={() => dispatch(authAction.logOut(navigation))}>
+          <Text style={styles.logOutTxt}>Logout</Text>
+        </TouchableOpacity>
+      ),
+    });
+  });
+
```

Comment:
Why this set options? why not directly have the logout in the config of the screen when declaring it?